### PR TITLE
Additional coverage for Unit Tests

### PR DESCRIPTION
**Pull request by IvoWams**
plus other minor things. I will expand this pullrequest at the end of the day (3 april 2020)

### CODE REVIEW

File: tests/unit/Service/ManagerTest.php [hunk 1]
```diff
@@ -306,6 +306,22 @@ public function testItCanConfigureAliases(): void
...
+
+    /**
+     * @depends testItCanConfigureAliases
+     *
+     * @return void
+     */
+    public function testICanConfigureIntersectingAliases(): void
+    {
+        try {
+        $this->tester->invokeMethod($this->manager, 'configureAliases', [[ 'foo_bar' => 'bar_foo' ]]);
```
Comment (IvoWams) [inline tests/unit/Service/ManagerTest.php:318]: indent

File: tests/unit/Service/ManagerTest.php [hunk 2]
```diff
@@ -351,6 +367,12 @@ public function testResolveInitializersThrowsAnExceptionWhenInitializerIsNotCall
         ]]);
     }
 
+    public function testResolveInvalidInitializersThrowsAnException(): void
```
Comment (IvoWams) [inline tests/unit/Service/ManagerTest.php:370]: missing phpdoc

File: tests/unit/Service/AbstractPluginManagerTest.php [hunk 3]
```diff
@@ -4,14 +4,13 @@
 
 namespace Tests\Unit\PayNL\Sdk\Service;
 
-use Codeception\{
-    Test\Unit as UnitTest,
+use Codeception\{Test\Unit as UnitTest,
```
Comment (isneep) [inline tests/unit/Service/AbstractPluginManagerTest.php:7]: Add newline after brace

File: tests/unit/Service/AbstractPluginManagerTest.php [hunk 4]
```diff
@@ -4,14 +4,13 @@
...
     Lib\ManagerTestTrait,
     TestAsset\Dummy,
+    TestAsset\DummyFilter,
     TestAsset\DummyInterface,
     TestAsset\DummyService,
-    TestAsset\SimpleModel
-};
+    TestAsset\SimpleModel};
```
Comment (isneep) [inline tests/unit/Service/AbstractPluginManagerTest.php:13]: Add newline before closing brace

File: tests/unit/Service/LoaderTest.php [hunk 5]
```diff
@@ -166,6 +176,11 @@ public function testItCanMergeServiceConfig(): void
         verify($serviceConfig->get('invokables'))->contains(Dummy::class);
     }
 
+    public function testItCanPreloadServiceConfig(): void
+    {
```
Comment (IvoWams) [inline tests/unit/Service/LoaderTest.php:180]: NYI

**Pull request merged**